permission_handler/example/lib/main.dart

```diff
@@ -1,127 +1,154 @@
-import 'dart:io';
-
 import 'package:flutter/material.dart';
-import 'package:permission_handler/permission_handler.dart';
+import 'package:flutter/widgets.dart';
+
+import 'template/globals.dart';
+
+void main() {
+  runApp(BaseflowPluginExample());
+}
 
-void main() => runApp(MyApp());
+/// Baseflow Plugin Example application
```

Comment:
Adhere to the Effective Dart style guidelines regarding to Doc comments, specifically rule "PREFER starting library or type comments with noun phrases." (https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-library-or-type-comments-with-noun-phrases)

```suggestion
  /// A Flutter application demonstrating the functionality of this plugin
```